package.json

```diff
@@ -4,7 +4,7 @@
   "author": "Freemius, Inc.",
   "license": "GPL-3.0",
   "homepage": "https://freemius.com",
-  "version": "2.3.2",
+  "version": "2.3.1",
```

Comment:
@fajardoleo needs an update to `2.4.0`. Also, please update all the `@version` of new methods added in this PR to `2.4.0` as well (that will be the new release version since it's a major one).